docs/usage/PRODUCT_SYNC.md

```diff
@@ -117,18 +117,9 @@ final ProductDraft productDraft =
         .build();
 ````
 
--  Some references in the product like `customerGroup` of a price and variant attributes with type `ReferenceType` do not support the `ResourceIdentifier` yet, 
+-  Some references in the product like variant attributes with type `ReferenceType` do not support the `ResourceIdentifier` yet, 
 for those references you have to provide the `key` value on the `id` field of the reference. This means that calling `getId()` on the reference should return its `key`. 
 
-````java
-final PriceDraft priceDraft = PriceDraftBuilder
```

Comment:
**This example should not be removed,** instead, you might update the customer group with `ResourceIdentifier.ofKey..`

I would move the sentence on line 120: `Some references...` with changing to, 

-  The product variant attributes with a type `ReferenceType` do not support the `ResourceIdentifier` yet,  for those references you have to provide the `key` value on the `id` field of the reference. This means that calling `getId()` on the reference should return its `key`.

Reply:
But this example is no longer related to `ReferenceType` mentioned. It applies `ResourceIdentifier` already.
Hence the example looks a bit irrelevant to line 120. wdyt?

Reply:
I was thinking that my comment is quite clear. Let me explain again:

The example code snippet should not be removed and should be updated with `ofKey` but the sentence above it would be better to move after this snippet example because it only explains the ReferenceType.

Reply:
It makes sense if we move the snippet before the sentence.
https://github.com/commercetools/commercetools-sync-java/pull/676/commits/3cd0d9bd7ed6751a200078273861479023747bb3